### ISSUE TITLE
Accessibility: two-finger scrolling on Android doesn't work as expected

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#10971]
I noticed a couple of issues:
* While scrolling the accessibility frame disappears in Flutter apps
* Sometimes, after two-finger scrolling no element is accessibility-selectable anymore (fixes itself after some more scrolling)

**Comment by goderbauer** [comment]
Issue 1 is caused by `RenderIgnorePointer` which is placed between the Scrollable and its content. As soon as scrolling starts, it is instructed to ignore all pointer events, which causes `RenderIgnorePointer` to also drop all of its children semantics on the floor. Hence, the node that was accessibility focused before scrolling started is removed from the semantics tree and can no longer be focused.

**Comment by goderbauer** [comment]
The second issue will be fixed by #11207.

**Comment by goderbauer** [comment]
Closing this and opening a separate issue for the remaining problem.
